libs/clients/icelandic-government-institution-vacancies/src/lib/apiConfig.ts

```diff
@@ -0,0 +1,44 @@
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import {
+  ConfigType,
+  LazyDuringDevScope,
+  XRoadConfig,
+} from '@island.is/nest/config'
+import { Configuration } from '../../gen/fetch'
+import { IcelandicGovernmentInstitutionVacanciesClientConfig } from './icelandicGovernmentInstitutionVacanciesClient.config'
+
+export const ApiConfig = {
+  provide: 'IcelandicGovernmentInstitutionVacanciesClientConfiguration',
+  scope: LazyDuringDevScope,
+  useFactory: (
+    xroadConfig: ConfigType<typeof XRoadConfig>,
+    config: ConfigType<
+      typeof IcelandicGovernmentInstitutionVacanciesClientConfig
+    >,
+  ) => {
+    const credentials = Buffer.from(
+      `${config.username}:${config.password}`,
+      'binary',
+    ).toString('base64')
+
+    return new Configuration({
+      fetchApi: createEnhancedFetch({
+        name: 'clients-icelandic-government-institution-vacancies',
+        logErrorResponseBody: true,
+        treat400ResponsesAsErrors: true,
+        timeout: 20000,
+      }),
+      basePath: `${xroadConfig.xRoadBasePath}/r1/${config.xRoadServicePath}`,
+      headers: {
+        'X-Road-Client': xroadConfig.xRoadClient,
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        Authorization: `Basic ${credentials}`,
+      },
```

Comment:
I believe that this is an older service which was semi recently added to xroad and has been used for this page here: https://starfatorg.is/ for a while now. However the goal is that starfatorg.is in the near future would point to island.is/starfatorg so that's why we are calling this service. Regarding why it is using basic auth or what system they are using for authentication is something I'm not sure about but I could contact someone to try and find out more if we want :)